app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -625,7 +625,8 @@ aside.onebox.stackexchange .onebox-body {
   }
 }
 
-// whitelistedgeneric twitter labels
+// allowlistedgeneric twitter labels
+.onebox.allowlistedgeneric,
 .onebox.whitelistedgeneric {
```

Comment:
Is this a line we can't remove yet?

Reply:
Yes we need to keep this, because onebox HTML structure is stored in post cooked content. We will need a full rebake before we can remove this. (Or a DB migration to find/replace cooked... but that sounds very risky)